nix/jupyter.nix

```diff
@@ -2,31 +2,36 @@
 
 let
   pkgs = import ./pkgs.nix;
-
+  compiler = "ghc8107";
   sources = import ./sources.nix { };
 
   # Jupyter - Nix integration #
   # See https://github.com/tweag/jupyterWith for standard Jupyter - Nix integration documentation
   # See https://github.com/tweag/jupyterWith/tree/master/kernels/ihaskell for iHaskell configuration in particular
 
   # Override the Haskell package set to use the current Funflow (and Kernmantle) code, rather than Hackage versions (so that tutorials use your current Funflow code)
+  # store/store-core are included here simply because the Github version has outpaced the Hackage version, and supports text >= 2.0
   # As those are not "nixified", use `callCabal2nix` to convert them
-  haskellPackages = pkgs.haskellPackages.override {
-    overrides = hself: hsuper: {
-      funflow = pkgs.haskell.lib.dontCheck
-        (hself.callCabal2nix "funflow" ../funflow { });
-      docker-client = pkgs.haskell.lib.dontCheck
-        (hself.callCabal2nix "docker-client" ../docker-client { });
-      kernmantle =
-        hself.callCabal2nix "kernmantle" "${sources.kernmantle}/kernmantle" { };
-      kernmantle-batteries = hself.callCabal2nix "kernmantle-batteries"
-        "${sources.kernmantle}/batteries" { };
-      kernmantle-caching =
-        hself.callCabal2nix "kernmantle-caching" "${sources.kernmantle}/caching"
-        { };
-    };
-  };
-
+  haskellPackages = pkgs.haskell.packages."${compiler}".override {
```

Comment:
@dorranh This was the big change to get the tutorials/pages to build (it's also hardwired to GHC 8.10.7, which `stack.yaml` and `shell.nix` use).